src/client/components/GamePage.js

```diff
@@ -447,35 +494,43 @@ export default () => {
 			// Restarts the countdown timer
 			setTurnCountdownTimer(SECONDS_PER_TURN)
 		})
-		state.socket.on('draw-card-received', (data) => {
+		state.socket.on('draw-card-received', (game) => {
 			dispatch({
 				type: 'SET_GAME',
 				payload: {
-					game: data.game,
+					game,
 				},
 			})
-		})
-		state.socket.on('card-invoke-received', (data) => {
+		}),
+			state.socket.on('card-burned', (game) => {
```

Comment:
several changed documents are not formatted properly. Be sure to apply prettier so they are aligned. Also attack a few screenshots for how the game will look like with this change